src/types.jl

```diff
@@ -414,8 +414,16 @@ AGray32(g::AbstractGray, alpha = 1) = AGray32(gray(g), alpha)
 # Note: with the exceptions of `alphacolor` and `coloralpha`, all
 # traits in the rest of this file are intended just for internal use
 
-const color3types = filter(x->(!x.abstract && length(fieldnames(x))>1), union(subtypes(Color), subtypes(AbstractRGB)))
-const parametric3 = filter(x->!isempty(x.parameters), color3types)
+const color3types = map(s->getfield(ColorTypes,s),
```

Comment:
There's a check for something being deprecated, which should be checked first. I thought I had done that. I guess not.